InformationScripting/src/nodes/CompositeQueryNode.h

```diff
@@ -39,6 +39,9 @@ class INFORMATIONSCRIPTING_API CompositeQueryNode : public Super<QueryNode>
 {
 	COMPOSITENODE_DECLARE_STANDARD_METHODS(CompositeQueryNode)
 	ATTRIBUTE(Model::TypedList<QueryNode>, queries, setQueries)
+
+	public:
```

Comment:
Wait, why are we implementing the visitor pattern in this way? We have the existing visitor class which would already work without the need for these accept methods. We should use that instead.